### PR TITLE
Jwilkes tenable

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CIAC-4058
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-89

## Description
Added the !tenable-io-get-asset-details command to be able to look up asset information and custom attributes.

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [X] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 1]
```diff
@@ -357,6 +360,20 @@ def send_asset_vuln_request(asset_id, date_range):
     return res.json()
 
 
+def send_asset_details_request(asset_id):
+    full_url = "{}workbenches/assets/{}/info".format(BASE_URL, asset_id)
+    res = requests.get(full_url, headers=AUTH_HEADERS, verify=USE_SSL)
+    res.raise_for_status()
+    return res.json()
```
Comment (omerKarkKatz) [inline Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:367]:
```suggestion
    full_url = "{}workbenches/assets/{}/info".format(BASE_URL, asset_id)
    try:
        res = requests.get(full_url, headers=AUTH_HEADERS, verify=USE_SSL)
        res.raise_for_status()
    except HTTPError as exc:
        return_error(f'Error calling for url {full_url}: error message {exc}')

    return res.json()

```

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 2]
```diff
@@ -357,6 +360,20 @@ def send_asset_vuln_request(asset_id, date_range):
...
+
+
+def send_asset_attributes_request(asset_id):
+    full_url = "{}api/v3/assets/{}/attributes".format(BASE_URL, asset_id)
+    res = requests.get(full_url, headers=AUTH_HEADERS, verify=USE_SSL)
+    res.raise_for_status()
+    return res.json()
```
Comment (omerKarkKatz) [inline Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:374]:
```suggestion
def send_asset_attributes_request(asset_id):
     full_url = "{}api/v3/assets/{}/attributes".format(BASE_URL, asset_id)
    try:
        res = requests.get(full_url, headers=AUTH_HEADERS, verify=USE_SSL)
        res.raise_for_status()
    except HTTPError as exc:
        return_error(f'Error calling for url {full_url}: error message {exc}')
    
    return res.json()
```

**Review by omerKarkKatz — COMMENTED** [review]
Hey @johnnywilkes, thanks a lot for your contribution looks great. 
I added some comments and suggestions below please look at them and fix accordingly.
If you have any further questions you can feel free to reach out here or by slack. 
When you finish let me know and we will schedule a short demo on your contribution. Thanks a lot!

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 3]
```diff
@@ -357,6 +360,20 @@ def send_asset_vuln_request(asset_id, date_range):
     return res.json()
 
 
+def send_asset_details_request(asset_id):
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:363]: @omerKarkKatz , none of the other function have doc strings.  Do you still want me to add? (not a problem either way)

**Comment by johnnywilkes** [comment]
@omerKarkKatz , I added types and docstrings, let me know if that works for you

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py]: added anyways, this should be resolved

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py [hunk 4]
```diff
@@ -462,6 +503,55 @@ def args_to_request_params(hostname, ip, date_range):
...
+    if not asset_id:
+        return CommandResults(readable_output=f'Asset not found: {ip}')
+
+    try:
+        info = send_asset_details_request(asset_id)
+        attrs = send_asset_attributes_request(asset_id)
+        if attrs:
+            attributes = []
+            for attr in attrs["attributes"]:
+                attributes.append({attr['name']: attr['value']})
```
Comment (omerKarkKatz) [inline Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:538]:
```suggestion
            attributes = []
            for attr in attrs.get("attributes", []):
                attributes.append({attr.get('name', ''): attr.get('value', '')})
```

Comment (johnnywilkes) [reply Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py:538]: makes sense

File: Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml [hunk 5]
```diff
@@ -425,7 +443,7 @@ script:
   script: '-'
   subtype: python3
   type: python
-  dockerimage: demisto/python3:3.10.6.33415
+  dockerimage: demisto/python3:3.10.8.37753
```
Comment (omerKarkKatz) [inline Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml:446]:
```suggestion
  dockerimage: demisto/python3:3.10.8.37753
```
```suggestion
  dockerimage: demisto/python3:3.10.8.39276
```

**Review by omerKarkKatz — APPROVED** [review]

**Pull request merged**